src/components/SearchByIDA/style.css

```diff
@@ -104,9 +105,33 @@ ul.ordered {
     //border: black solid 1px;
     width: 1em;
   }
-  @nest & button:hover {
+  @nest & button.drag {
```

Comment:
I suggest putting draggable sections on both the beginning and the end of the domain graphic. I would also consider centre justifying the text in the domain graphic or increasing the amount of padding between the draggable section and the domain text.